ui/components/component-library/banner-tip/banner-tip.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import {
+  AlignItems,
+  BorderColor,
+  DISPLAY,
+} from '../../../helpers/constants/design-system';
+import Box from '../../ui/box';
+import { BannerBase } from '..';
+import { BannerTipLogoType } from './banner-tip.constants';
+
+export const BannerTip = ({
+  children,
+  className,
+  logoType = BannerTipLogoType.greeting,
+  logoWrapperProps,
+  logoProps,
+  ...props
+}) => {
+  return (
+    <BannerBase
+      startAccessory={
+        <Box
+          display={DISPLAY.FLEX}
+          alignItems={AlignItems.center}
+          {...logoWrapperProps}
+        >
+          <Box
+            as="img"
+            className="mm-banner-tip--logo"
+            src={`images/fox-${logoType}.png`}
+            alt={logoType}
+            {...logoProps}
+          />
+        </Box>
+      }
```

Comment:
Could we make this more flexible by allowing an over ride? I may have got the syntax wrong here so maybe update manually

```suggestion
      startAccessory={ startAccessory ? startAccessory :
        <Box
          display={DISPLAY.FLEX}
          alignItems={AlignItems.center}
          {...logoWrapperProps}
        >
          <Box
            as="img"
            className="mm-banner-tip--logo"
            src={`images/fox-${logoType}.png`}
            alt={logoType}
            {...logoProps}
          />
        </Box>
      }
```